File: 프로그래머스/0/120826. 특정 문자 제거하기/특정 문자 제거하기.py
import re 

def solution(my_string, letter):
    return re.sub(letter, '', my_string)

'''
re.sub('[1-9]', '', "BCBd1112344be")
re.sub('[a-zA-Z]', '', "BCBd1112344be")
re.sub('[a-z]', '', "BCBd1112344be")
'''

File: 프로그래머스/0/120826. 특정 문자 제거하기/특정 문자 제거하기.js
function solution(my_string, letter) {
    // 1
    // return my_string.replaceAll(letter, '');
    
    // 2
    let reg = new RegExp(letter, 'g');
    return my_string.replace(reg, '');
}

// "BCB234235678678zdfbdbe".replace(/B/g, '')
// "BCB234235678678zdfbdbe".replace(/[123]/g, '')
// "BCB234235678678zdfbdbe".replace(/[1-9]/g, '')
// "BCB234235678678zdfbdbe".replace(/[a-z]/g, '')
// "BCB234235678678zdfbdbe".replace(/[A-Z]/g, '')